WoT Week Planning Folder (#586)
w3c/wot-testing: events/2024.11.Munich/Readme.md

# WoT Week at Munich

> The logistics can be found at https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning .

Reviewer: ```suggestion
The logistics can be found at https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning.
```

In this folder, technical information is found such as test plans of features.

## WIP - User Scenarios

Different tests we do can be placed into context, e.g. smart cities. 
This way, once we present WoT to the visitors, they can relate better.
Some ideas:

* Smart City
* WoT for Cloud Systems

## WIP - TD Topics

### Making Sure

* Additional Responses
  * Error Handling on the application level (Scripting API is connected here)
  * Multiple possible successful responses
  * Default behavior
* Linking between TDs

> * Specifying multiple types (at types) to TDs: This is confusing to people. We need more guidance.

Reviewer: Not sure what is meant with "(at types)" ?

Author: ```suggestion
* Specifying multiple `@type`s to TDs: This is confusing to people. We need more guidance.
```

* Complex actions: Async and sync

> * Meta Operations (top-level form operations):Are they implemented correctly

Reviewer: ```suggestion
* Meta Operations (top-level form operations): Are they implemented correctly
```
